.github/workflows/benchmark.yml

```
@@ -19,7 +19,7 @@ jobs:
       MAIN-BENCH-20: ${{ steps.benchmark-main.outputs.BENCH_RESULT20 }}
```

The benchmark workflows have a few more places where node 22 should be added, and <=16 should be removed

```suggestion
      MAIN-BENCH-20: ${{ steps.benchmark-main.outputs.BENCH_RESULT20 }}
      MAIN-BENCH-22: ${{ steps.benchmark-main.outputs.BENCH_RESULT22 }}
```